boltzrpc/boltzrpc.proto

```diff
@@ -592,9 +607,10 @@ message LightningChannel {
 message SwapStats {
     uint64 total_fees = 1;
     uint64 total_amount = 2;
-    uint64 avg_fees = 3;
-    uint64 avg_amount = 4;
+    uint64 avg_fees = 3 [deprecated = true];
+    uint64 avg_amount = 4 [deprecated = true];
```

Comment:
Why are those deprecated?

Reply:
initially did it because they are just derived from the other ones and a bit of unnecessary. but we could still keep them